search-app/src/cmr/search/services/query_execution/facets/hierarchical_v2_facets.clj

```diff
@@ -342,11 +412,42 @@
       (apply concat terms (for [child-facet (:children facet)]
                             (get-terms-at-depth child-facet (dec depth) terms))))))
 
+(defn get-depth-of-field
+  "Use recursion to walk through the facet hierarchy to find the passed in field and return its
+  position in the hierarchy. Returns a nested list of values. If its not found then return nil."
+  [facet field count]
```

Comment:
We shouldn't shadow the `count` function, so `depth` or similar would be preferred

Reply:
I changed count to depth.